### PR TITLE
[MRG] apply_forward_raw takes info instead of raw_template.

### PR DESCRIPTION
**Pull request by jaeilepp**
Closes https://github.com/mne-tools/mne-python/issues/2493

### CODE REVIEW

File: mne/forward/forward.py [hunk 1]
```diff
@@ -1230,28 +1228,31 @@ def apply_forward_raw(fwd, stc, raw_template, start=None, stop=None,
     --------
     apply_forward: Compute sensor space data and return an Evoked object.
     """
+    if isinstance(info, _BaseRaw):
+        warnings.warn('The "Raw_template" parameter is being deprecated '
+                      'and will be removed in MNE-0.11. '
+                      'Please provide info parameter instead',
+                      DeprecationWarning)
```
Comment (jaeilepp) [inline mne/forward/forward.py]: Not sure if it's okay to deprecate like this, but it wont break people's code unless they give the parameter explicitly (``raw_template=raw``).

Comment (agramfort) [inline mne/forward/forward.py]: to handle this you can add a last parameter raw_template=None and capture this.

File: mne/forward/forward.py [hunk 2]
```diff
@@ -1230,28 +1228,31 @@ def apply_forward_raw(fwd, stc, raw_template, start=None, stop=None,
...
-                             'raw_template.' % ch_name)
+                             'info.' % ch_name)
 
     # project the source estimate to the sensor space
     data, times = _apply_forward(fwd, stc, start, stop)
 
-    # store sensor data in Raw object using the template
-    raw = raw_template.copy()
+    sfreq = 1.0 / stc.tstep
+    info = _fill_measurement_info(info, fwd, sfreq)
```
Comment (agramfort) [inline mne/forward/forward.py]: _fill_measurement_info makes a copy? there should be not side effect in passed parameter.

**Comment by agramfort** [comment]
besides LGTM

File: mne/forward/forward.py (hunk 2 above)
Comment (jaeilepp) [inline mne/forward/forward.py]: Yes. _fill_measurement_info calls pick_info which makes a deepcopy.

File: mne/forward/forward.py (hunk 1 above)
Comment (jaeilepp) [inline mne/forward/forward.py]: Yeah, but since ``info`` is a required parameter, the call to this function would fail before capturing this case.

Comment (agramfort) [inline mne/forward/forward.py]: good point.

**Comment by agramfort** [comment]
thanks @jaeilepp !

**Pull request merged**